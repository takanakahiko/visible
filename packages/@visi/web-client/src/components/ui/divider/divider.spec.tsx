import { render } from '@testing-library/react';
import React from 'react';

import { Divider } from '.';

describe('Divider', () => {
  it('matches snapshot', () => {
    const { container } = render(<Divider />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
