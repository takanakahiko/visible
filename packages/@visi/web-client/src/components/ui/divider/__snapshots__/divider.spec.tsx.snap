// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider matches snapshot 1`] = `
<hr
  class="block my-8 border bt-2 border-gray-300 w-full"
/>
`;
